app/views/articles/show.html.erb

```diff
@@ -85,16 +132,16 @@
     <% if @article.video_state == "PROGRESSING" %>
       <h1 style="text-align:center;">⏳ Video Transcoding In Progress ⏳</h1>
     <% end %>
-    <article itemscope itemtype="http://schema.org/Article" itemprop="mainEntity">
-      <meta itemprop="url" content="<%= article_url(@article) %>">
-      <meta itemprop="image" content="<%= article_social_image_url(@article) %>">
-      <div itemprop="publisher" itemscope itemtype="https://schema.org/Organization">
-        <div itemprop="logo" itemscope itemtype="https://schema.org/ImageObject">
-          <meta itemprop="url" content="<%= cloudinary(SiteConfig.logo_png, 192, "png") %>">
-          <meta itemprop="width" content="192">
-          <meta itemprop="height" content="192">
+    <article>
+      <meta content="<%= article_url(@article) %>">
+      <meta content="<%= article_social_image_url(@article) %>">
+      <div>
+        <div>
+          <meta content="<%= cloudinary(SiteConfig.logo_png, 192, "png") %>">
+          <meta content="192">
+          <meta content="192">
         </div>
-        <meta itemprop="name" content="<%= community_qualified_name %>">
+        <meta content="<%= community_qualified_name %>">
```

Comment:
same as for the video article. I don't think we need these meta fields anymore as they don't have a property